packages/orchestration/src/orchestration-types.ts

```diff
@@ -432,17 +434,37 @@ interface TranslationConfigParameters {
    * @example targetLanguage: 'en-US'
    */
   targetLanguage: string;
+  /**
+   * List of selectors to apply translation to.
+   */
+  applyTo?: SAPDocumentTranslationApplyToSelector[];
+}
+
+/**
+ * Output parameters for translation configuration.
+ */
+interface TranslationConfigParametersOutput {
+  /**
+   * Language of the text to be translated.
+   * @example sourceLanguage: 'de-DE'
+   */
+  sourceLanguage?: string;
+  /**
+   * Language to which the text should be translated.
+   * @example targetLanguage: 'en-US'
+   */
+  targetLanguage: SAPDocumentTranslationOutputTargetLanguage;
```

Comment:
Same, if you would like to support the selector in this PR already, changeset + documentation.

Otherwise, change it back to string + create follow-up BLI.